### PR TITLE
Cleanup Console.WriteLine()

### LINKED ISSUE #28358: Strange messages in System.Xml.Xsl.XslCompiledTransformApi.Tests log
System.Xml.Xsl.XslCompiledTransformApi.Tests log contains next messages in log:
```
    Starting:    System.Xml.Xsl.XslCompiledTransformApi.Tests
  Message #2
  Message #4
  <Message>Message #6</Message>
  <Message>Message #8</Message>
    Finished:    System.Xml.Xsl.XslCompiledTransformApi.Tests
```

### PR DESCRIPTION
**Pull request by MarcoRossignoli**
Closes #28358

### CODE REVIEW

File: src/System.Private.Xml/src/System/Xml/Xsl/XsltOld/MessageAction.cs [hunk 1]
```diff
@@ -56,8 +56,7 @@ internal override void Execute(Processor processor, ActionFrame frame)
 
                 case ProcessingChildren:
                     TextOnlyOutput recOutput = processor.PopOutput() as TextOnlyOutput;
-                    Debug.Assert(recOutput != null);
-                    Console.WriteLine(recOutput.Writer.ToString());
```
Comment (MarcoRossignoli) [inline src/System.Private.Xml/src/System/Xml/Xsl/XsltOld/MessageAction.cs]: Doesn't show nothing on console during tests, but i think it's typo.

File: src/System.Private.Xml/tests/Writers/RwFactory/CXmlDriverVariation.cs [hunk 2]
```diff
@@ -58,8 +58,7 @@ public override tagVARIATION_STATUS Execute()
                 res = (tagVARIATION_STATUS)HandleException(e);
             }
             catch (Exception e)
-            {
-                Console.WriteLine(e);
```
Comment (MarcoRossignoli) [inline src/System.Private.Xml/tests/Writers/RwFactory/CXmlDriverVariation.cs]: Doesn't show nothing on console during tests, but i think it's typo.

File: src/System.Private.Xml/src/System/Xml/Xsl/Runtime/XmlQueryContext.cs [hunk 3]
```diff
@@ -328,9 +328,7 @@ public void OnXsltMessageEncountered(string message)
             XsltMessageEncounteredEventHandler onMessage = (_argList != null) ? _argList.xsltMessageEncountered : null;
 
             if (onMessage != null)
-                onMessage(this, new XmlILQueryEventArgs(message));
-            else
-                Console.WriteLine(message);
```
Comment (MarcoRossignoli) [inline src/System.Private.Xml/src/System/Xml/Xsl/Runtime/XmlQueryContext.cs]: Show on console during tests.

**Comment by stephentoub** [comment]
Thanks

**Pull request merged**